client/chat/settings/general.tsx

```diff
@@ -0,0 +1,129 @@
+import { debounce } from 'lodash-es'
+import React, { useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+import { EditChannelRequest, SbChannelId } from '../../../common/chat'
+import { useForm } from '../../forms/form-hook'
+import { MaterialIcon } from '../../icons/material/material-icon'
+import { TextField } from '../../material/text-field'
+import { useAppDispatch } from '../../redux-hooks'
+import { useStableCallback } from '../../state-hooks'
+import { colorError, colorSuccess } from '../../styles/colors'
+import { subtitle1 } from '../../styles/typography'
+import { updateChannel } from '../action-creators'
+
+const ErrorText = styled.div`
+  ${subtitle1};
+  color: ${colorError};
+  margin-bottom: 8px;
+`
+
+const SuccessIcon = styled(MaterialIcon).attrs({ icon: 'check_circle' })`
+  color: ${colorSuccess};
+`
+
+enum UpdateStatus {
+  Updating = 'Updating',
```

Comment:
zero need for these to be strings, just set no value on the enum. We only use string enums for stuff that needs to be maintained across versions or communicated across processes, these are not that